Remove nesting (#219)
ietf-wg-aipref/drafts: draft-ietf-aipref-vocab.md

as it relates to understanding what the preferences for a given asset are
(as opposed to what actions might then follow).

A recipient can only apply preferences it understands.

> Recipients might not implement this specification.

Reviewer: Hmm. This may raise eyebrows.

Reviewer 2: I don't quite understand why we need this text. Isn't that always true?

Author: ```suggestion
```

Yeah, no need.

## Vocabulary Extensions {#vocab-extension}

Extensions to this vocabulary are defined
in a standards-track RFC that updates this document.

The definition of the extension MUST define

> how any potential overlap is resolved.

Author: ```suggestion
The definition of the extension MUST define
how any potential overlap between usage categories is resolved.
Definitions can identify which usage category applies
for any such overlap.
```

This document resolves overlap between categories of use
by explicitly designating any usage
that might be covered by multiple terms
as belonging to a single category.

> This approach is strongly encouraged for any extension.

Author: ```suggestion
how any potential overlap between usage categories is resolved.
Definitions can identify which usage category applies
when such an overlap occurs.
```

## Vocabulary Extensions {#vocab-extension}

> Extensions to this vocabulary MUST be defined

Author: ```suggestion
Extensions to this vocabulary are defined
```

in a standards-track RFC that updates this document.

Vocabulary extensions MUST consider

> how the definition of terms could overlap with other existing terms.

Author: ```suggestion
```

Definitions can identify which usage category applies
for any such overlap.

This document resolves overlap between categories of use
by explicitly designating any usage
that might be covered by multiple terms
as belonging to a single category.
This approach is strongly encouraged for any extension.

> 

Author: ```suggestion
```

## Vocabulary Extensions {#vocab-extension}

Extensions to this vocabulary MUST be defined

> in a standards-track RFC that updates this document.

Reviewer 2: I understand what you are doing here, but why not just say "are defined" rather than trying to cram in 2119 keywords.

Author: I believe that the usual model for extensions is that an extension doesn't generally have to update the document that defined the extension point.  In this case, the agreement was to require an RFC.  Still, I'm not a fan of overuse of MUST.

Vocabulary extensions MUST consider

> how the definition of terms could overlap with other existing terms.

Reviewer 2: I would remove this. I mean you should do this but it's not really actionable unless you are requiring a section in the extensions document. Otherwise you are saying extension authors need to consider.

The definition of the extension MUST define
how any potential overlap is resolved.

This document resolves overlap between categories of use
by explicitly designating any usage
that might be covered by multiple terms

> as belonging to a single category.

Reviewer: This is a bit abstract

Reviewer 2: Agreed.

This approach is strongly encouraged for any extension.

Systems that use this vocabulary might seek to integrate
the terms in this vocabulary

> as part of a larger data model.

Reviewer: ```suggestion
as part of a larger data model that includes other terms not defined here.
```

> This is not subject to the RFC requirement above,

Reviewer: ```suggestion
Such usage is not subject to the RFC requirement above,
```
